contrib/completion/bash/docker

```diff
@@ -246,9 +263,6 @@ _docker_docker() {
 		-*)
 			COMPREPLY=( $( compgen -W "$boolean_options $main_options_with_args" -- "$cur" ) )
```

Comment:
`$main_options_with_args` does not belong here as it contains (part of the) global flags like `-D`.
Please remove the non-global flags from `$main_options_with_args` below (see comment above) and insert them here literally or via a local variable.